src/account/reducer.ts

```diff
@@ -16,7 +17,7 @@ export interface State {
   devModeClickCount: number
   photosNUXClicked: boolean
   pincodeType: PincodeType
-  backupCompleted: boolean
+  backupCompleted: boolean // recovery phrase backup
```

Comment:
this seems redundant with keyless backup state, which contains the status of the backup. it also just seems non-intuitive to me to keep keyless backup state in two different places. Can you explain why you decided to add it here?

Reply:
the entire keylessBackup field is blacklisted so we don't persist sensitive info, so we need to track it in a separate place. It's possible to blacklist a subset of fields, but that requires a significant refactor in our reducer setup. 

> this seems redundant with keyless backup state

I believe we need two statuses because this will be true for existing users who have done the backup quiz, but we still want to show the "Set up" option for cloud backup. Although I guess if someone finishes cloud backup, maybe there shouldn't be an option to do the backup quiz at all?

Reply:
no I think there should still be an option

Reply:
more backups is good

Reply:
I left this comment on the wrong place, meant it to be on `cloudBackupCompleted`

Reply:
> the entire keylessBackup field is blacklisted so we don't persist sensitive info

I didn't realize this so I'm marking as resolved now